### PR TITLE
Ledger transaction help icon

### PR DESCRIPTION
**Pull request by TingALin**
HelpIcons next to the rejected Ledger transaction pending message that open the help overlay states

addresses  #819

### CODE REVIEW

File: src/library/SubmitTx/Wrappers.tsx [hunk 1]
```diff
@@ -37,8 +37,10 @@ export const Wrapper = styled.div<{ noMargin?: boolean }>`
       align-items: center;
 
       p {
+        display: flex;
+        flex-direction: row;
         color: var(--text-color-secondary);
-        font-size: 1rem;
+        font-size: 1.1rem;
```
Comment (rossbulat) [inline src/library/SubmitTx/Wrappers.tsx:43]:
```suggestion
        font-size: 1rem;
```

File: src/library/SubmitTx/ManualSign.tsx [hunk 2]
```diff
@@ -139,6 +141,14 @@ export const ManualSign = ({
                   ? fallbackMessage
                   : statusCodeTitle)
               : fallbackMessage}
+            {defaultMessage ===
```
Comment (rossbulat) [inline src/library/SubmitTx/ManualSign.tsx:144]: There should be `helpKey` returned by determineStatusCode, and that helpKey should be tested here.

**Review by rossbulat — CHANGES_REQUESTED** [review]
An additional `helpKey` needs to be returned from determineStatusCode.

File: src/library/SubmitTx/Wrappers.tsx [hunk 3]
```diff
@@ -37,6 +37,7 @@ export const Wrapper = styled.div<{ noMargin?: boolean }>`
       align-items: center;
 
       p {
+        display: flex;
         color: var(--text-color-secondary);
```
Comment (rossbulat) [inline src/library/SubmitTx/Wrappers.tsx:41]: Move this variable to the top of this class.

File: src/config/help.ts [hunk 4]
```diff
@@ -19,6 +27,7 @@ export const HelpConfig: HelpItems = [
       'Ideal Staked',
       'Supply Staked',
       'Ledger Hardware Wallets',
```
Comment (rossbulat) [inline src/config/help.ts:29]: These should be removed as they have been moved under the `ledger` key.

File: src/config/help.ts [hunk 5]
```diff
@@ -4,6 +4,14 @@
 import type { HelpItems } from 'contexts/Help/types';
 
 export const HelpConfig: HelpItems = [
+  {
+    key: 'ledger',
+    definitions: [
+      'Ledger Hardware Wallets',
+      'Ledger Rejected Transaction Options',
```
Comment (rossbulat) [inline src/config/help.ts:11]: Rename to `Ledger Rejected Transaction`.

File: src/locale/cn/help.json [hunk 6]
```diff
@@ -136,6 +136,10 @@
           "详情请在其官方网站上查找:ledger.com."
         ]
       ],
+      "ledgerRejectedTransactionOptions": [
```
Comment (rossbulat) [inline src/locale/cn/help.json:139]: With the above rename this will be `ledgerRejectedTransaction`

File: src/locale/en/help.json [hunk 7]
```diff
@@ -145,6 +145,10 @@
           "Find more information about Ledger at their official website, ledger.com."
         ]
       ],
+      "ledgerRejectedTransactionOptions": [
```
Comment (rossbulat) [inline src/locale/en/help.json:148]: With the above rename this will be `ledgerRejectedTransaction`.

File: src/locale/en/help.json [hunk 8]
```diff
@@ -145,6 +145,10 @@
...
+        "Ledger Rejected Transaction Options",
+        ["You can either sign the transaction again or close the window"]
```
Comment (rossbulat) [inline src/locale/en/help.json:150]: Missing full stop, `.`.

**Comment by TingALin** [comment]
> In addition to the comments, `defaultMessage` is still not `helpKey` supported. I also expect more new `helpKeys` to explain the various states of the Ledger device.

Considering defaultMessage is wherever StatusCode is , I think it's more reasonable to just have the helpKey in defaultMessage, and leave the StatusCode the way it was, but for now, both of the states support helpKey

**Comment by rossbulat** [comment]
⚠️ master -> main branch migration ⚠️
Polkadot staking dashboard's master branch has been renamed to main. If developing locally, remember to update .git/config and fetch main to get the latest default branch.

**Comment by rossbulat** [comment]
I just need to expand on the help content. After which @TingALin please translate into CN. Will let you know when the EN versions are pushed.

File: src/locale/cn/help.json [hunk 9]
```diff
@@ -136,6 +136,20 @@
           "详情请在其官方网站上查找:ledger.com."
         ]
       ],
+      "ledgerRejectedTransaction": [
+        "Ledger拒绝交易",
+        [
+          "The transaction was rejected on the Ledger device and will not be submitted.",
```
Comment (rossbulat) [inline src/locale/cn/help.json:142]: @TingALin translations needed.

File: src/locale/cn/help.json [hunk 10]
```diff
@@ -136,6 +136,20 @@
...
+          "Submit a transaction by approving it on the Ledger device."
+        ]
+      ],
+      "ledgerRequestTimeout": [
+        "Ledger请求超时",
+        [
+          "Timeout errors can occur when the Ledger device is busy, or when an operation fails to complete in time.",
```
Comment (rossbulat) [inline src/locale/cn/help.json:149]: @TingALin translations needed.

File: src/locale/cn/help.json [hunk 11]
```diff
@@ -199,6 +213,13 @@
           "该值和验证人的自我抵押一起形成验证人总抵押数,值得注意的是,该值因为随着提名人在该Session的质押金额给活跃验证人的重新分配,每一个era都会发生变化."
         ]
       ],
+      "openAppOnLedger": [
```
Comment (rossbulat) [inline src/locale/cn/help.json:216]: @TingALin translations needed.

File: src/locale/cn/help.json [hunk 12]
```diff
@@ -300,6 +321,13 @@
           "作为提名人，您可以选择支持哪个验证人并获得奖励."
         ]
       ],
+      "wrongTransaction": [
```
Comment (rossbulat) [inline src/locale/cn/help.json:324]: @TingALin translations needed.

**Review by rossbulat — COMMENTED** [review]
Just translations needed now.

**Review by rossbulat — APPROVED** [review]

**Pull request merged**